src/win/win_serial_passthrough.c

```diff
@@ -161,7 +161,8 @@ static int
 open_pseudo_terminal(serial_passthrough_t *dev)
 {
     char ascii_pipe_name[1024] = { 0 };
-    snprintf(ascii_pipe_name, sizeof(ascii_pipe_name), "\\\\.\\pipe\\86Box\\%s", vm_name);
+    if (dev->named_pipe[0] != 0) strncpy(ascii_pipe_name, dev->named_pipe, 1023);
+    else snprintf(ascii_pipe_name, sizeof(ascii_pipe_name), "\\\\.\\pipe\\86Box\\%s", vm_name);
```

Comment:
Personally I think the user should have to type out the full path at all times.

Adding a non-obvious prefix if it's not a full path is pretty confusing. We should just error out if the provided string is not a valid named pipe path.